calico/etcddriver/driver.py

```diff
@@ -78,6 +80,8 @@
 
 # Threshold in seconds for detecting watcher tight looping on exception.
 REQ_TIGHT_LOOP_THRESH = 0.2
+# How often to log stats.
+STATS_LOG_INTERVAL = 10
```

Comment:
It does trigger even without load.  I'm happy to bump it up to, say 60s, if you think that's better.